### PR TITLE
update hwameistor docs

### PR DESCRIPTION
**Pull request by Jeanine-tw**
fix issues:
#2687 
#2688 
#2689

### CODE REVIEW

File: docs/en/docs/storage/hwameistor/install/post-check.md [hunk 1]
```diff
@@ -1,103 +1,101 @@
...
+localvolumes                         lv                           hwameistor.io/v1alpha1   false        LocalVolume
+localvolumesnapshotrestores          lvsrestore,lvsnaprestore     hwameistor.io/v1alpha1   false        LocalVolumeSnapshotRestore
+localvolumesnapshots                 lvs                          hwameistor.io/v1alpha1   false        LocalVolumeSnapshot
+resizepolicies                                                    hwameistor.io/v1alpha1   false        ResizePolicy
 ```
 
-## Check `LocalDiskNode` and `localDisks`
+For the details about CRDs, please also refer to [CRDs](../../architecture/apis.md).
+
+## Check `LocalDiskNodes` and `localDisks`
```
Comment (SSmallMonster) [inline docs/en/docs/storage/hwameistor/install/post-check.md:73]:
```suggestion
## Check `LocalDiskNodes` and `LocalDisks`
```

File: docs/en/docs/storage/hwameistor/install/post-check.md [hunk 2]
```diff
@@ -1,103 +1,101 @@
...
-!!! info
+> The components of `local-disk-manager` and `local-storage` are `DaemonSets`, and should have one Pod on each Kubernetes node.
 
-    `local-disk-manager` and `local-storage` are `DaemonSet`. There should be a DaemonSet Pod on each Kubernetes node.
+## Check APIs
 
-## Check HwameiStor CRDs (i.e. APIs)
-
-The following HwameiStor Custom Resource Definitions (CRDs) must be installed on the system.
+HwameiStor CRDs create the following APIs.
```
Comment (SSmallMonster) [inline docs/en/docs/storage/hwameistor/install/post-check.md:46]:
```suggestion
HwameiStor create the following CRDs.
```

File: docs/en/docs/storage/hwameistor/install/post-check.md [hunk 3]
```diff
@@ -1,103 +1,101 @@
...
+hwameistor-scheduler-6db69957f-r58j6                       1/1     Running   0             23m
+hwameistor-ui-744cd78d84-vktjq                             1/1     Running   0             23m
+hwameistor-volume-evictor-5db99cf979-4674n                 1/1     Running   0             23m
 ```
 
-!!! info
+> The components of `local-disk-manager` and `local-storage` are `DaemonSets`, and should have one Pod on each Kubernetes node.
 
-    `local-disk-manager` and `local-storage` are `DaemonSet`. There should be a DaemonSet Pod on each Kubernetes node.
+## Check APIs
```
Comment (SSmallMonster) [inline docs/en/docs/storage/hwameistor/install/post-check.md:44]:
```suggestion
## Check CRDs
```

**Review by windsonsea — APPROVED** [review]

File: docs/en/docs/storage/hwameistor/install/post-check.md (hunk 1 above)
Comment (Jeanine-tw) [reply docs/en/docs/storage/hwameistor/install/post-check.md]: done

**Pull request merged**